Makefile

```diff
@@ -1,7 +1,7 @@
 build:
 	git clone -b 0.7.0 https://github.com/ONSdigital/sdx-common.git
 	pip3 install ./sdx-common
```

Comment:
Are you removing sdx-common from the makefile as part of this ticket? You've made the changes in sdx-sequence.